test/JDBC/expected/cast_eliminate-vu-verify.out

```diff
@@ -0,0 +1,82 @@
+set babelfish_showplan_all on
```

Comment:
Before this, let's do `analyze` manually because auto analyze can introduce flaky results.